raiden/api/python.py

```diff
@@ -419,6 +420,78 @@ def channel_open(
 
         return channel_state.identifier
 
+    def set_total_channel_withdraw(
+        self,
+        registry_address: typing.PaymentNetworkAddress,
+        token_address: typing.TokenAddress,
+        partner_address: typing.Address,
+        total_withdraw: typing.WithdrawAmount,
+        retry_timeout: typing.NetworkTimeout = DEFAULT_RETRY_TIMEOUT,
+    ):
+        """ Set the `total_withdraw` in the channel with the peer at `partner_address` and the
+        given `token_address.
+
+        Raises:
+            InvalidAddress: If either token_address or partner_address is not
+                20 bytes long.
+            TransactionThrew: May happen for multiple reasons:
+                - If the token approval fails, e.g. the token may validate if
```

Comment:
this comment is still referring to `deposit` in a couple of places. can you rework it?